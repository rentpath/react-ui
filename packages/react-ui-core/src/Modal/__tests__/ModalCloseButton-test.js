import React from 'react'
import renderer from 'react-test-renderer'
import ThemedCloseButton from '../ModalCloseButton'

const ModalCloseButton = ThemedCloseButton.WrappedComponent

describe('ModalCloseButton', () => {
  it('renders default children if none passed', () => {
    const snap = renderer
      .create(<ModalCloseButton />)
      .toJSON()
    expect(snap).toMatchSnapshot()
  })

  it('renders children passed', () => {
    const snap = renderer
      .create(<ModalCloseButton>Close</ModalCloseButton>)
      .toJSON()

    expect(snap).toMatchSnapshot()
  })
})
