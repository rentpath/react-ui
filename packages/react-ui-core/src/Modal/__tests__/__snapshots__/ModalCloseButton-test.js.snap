// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalCloseButton renders children passed 1`] = `
<div
  className=""
  data-tid="modal-close-button"
>
  Close
</div>
`;

exports[`ModalCloseButton renders default children if none passed 1`] = `
<div
  className=""
  data-tid="modal-close-button"
>
  X
</div>
`;
